code/__DEFINES/mobs.dm

```diff
@@ -282,6 +282,17 @@
 //Used as an upper limit for species that continuously gain nutriment
 #define NUTRITION_LEVEL_ALMOST_FULL 535
 
+#define CRYSTALIZE_COOLDOWN_LENGTH 120 SECONDS
+#define CRYSTALIZE_PRE_WAIT_TIME 40 SECONDS
+#define CRYSTALIZE_DISARM_WAIT_TIME 120 SECONDS
+#define CRYSTALIZE_HEAL_TIME 60 SECONDS
```

Comment:
This is so incredibly short. The cooldown time should be like 10-15 minutes minimum and and heal time should be longer imo.

Reply:
They are not capable of taking any medical care when they're in a crystal if I am right. The time can be a bit longer but 10-15 mins seems excessive.
They should be revived as 'going to die soon' as being at 1~5% full hp instead being revived as full condition.

Reply:
Although they get brain trauma upon revival though..

Reply:
5 minutes sounds fine, but seems like a bit long considering we prevent them from ghosting during that time

Reply:
How about 3 minutes then?